colossalai/inference/server/completion_service.py

```diff
@@ -1,10 +1,18 @@
 import asyncio
+from typing import AsyncIterator
 
 from colossalai.inference.core.async_engine import AsyncInferenceEngine
 
 from .utils import id_generator
 
 
+async def completion_stream_generator(
+    result_generator: AsyncIterator,
```

Comment:
Is this function essential? It seems that it just create a new async generator from an existing async generator

Reply:
It is needed as we should return a async generator when using streaming response, while not, we return a single  content.(which is also `generator` in this case).